build.sh

```diff
@@ -187,6 +187,10 @@ test_coverage() {
     fi
 
     reportgenerator -reports:tests/**/*.cobertura.xml -targetdir:tests/Coverage -reporttypes:"Cobertura"
+
+    newName="Cobertura-${OS}.xml"
```

Comment:
`${OS}` results in

| OS | result |
| -- | -- |
| Windows | `stored report to tests/Coverage/Cobertura-Windows_NT.xml` |
| Linux | `stored report to tests/Coverage/Cobertura-.xml` |
| Mac | `stored report to tests/Coverage/Cobertura-.xml` |

Maybe `${uname}` is better.

Reply:
With `$(uname)`:

| OS | result |
| -- | -- |
| Windows | `stored report to tests/Coverage/Cobertura-MINGW64_NT-10.0-17763.xml` |
| Linux | `stored report to tests/Coverage/Cobertura-Linux.xml` |
| Mac | `stored report to tests/Coverage/Cobertura-Darwin.xml` |

But each job uploads it's results, so they get overwritten. I think it's better to employ something like https://github.com/gfoidl-Tests/Native_Managed/blob/3e7f9d632b886e5ca93f16cedebdd1a32d5d396e/.azure/pipelines/jobs/build_and_test.yml#L17-L39